charts/nublado2/templates/nublado_config.yaml

```diff
@@ -0,0 +1,9 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: nublado-config
+  labels:
+    {{- include "nublado2.labels" . | nindent 4 }}
+data:
+  hub_config.yaml: |
+    {{- toYaml .Values.nublado_config | nindent 4 }}
```

Comment:
The `nublado` here seems very redundant. Why not just `.Values.config`? Or if you want to have parallel structure between `jupyterlab.config` and this chart, `nublado.config` would be better so that the file is structured like:
```
jupyterlab:
  proxy:
    # ...
nublado:
  config:
    # ...
```

Reply:
I'm confused by the mention of parallel structure, since all the keys would need to be under nublado anyway.  I'm fine with whatever name we want, just config seemed a bit ambiguous.  I can pick any name we want.

Reply:
Sounds good to me!